source/emit.ts

```diff
@@ -149,8 +149,8 @@ export async function emitToString(
       ] as const;
     }
 
-    export type Type = ReturnType<typeof getTypes>[number];
-    export type TypeName = Type["name"];
+    export type TypeRuntimeInformation = ReturnType<typeof getTypes>[number];
```

Comment:
I like runtime. But maybe have just `Runtime`as a prefix to them all? Shorter names, makes them easier to find "grouped" in the intellisense? It is it more important that they are grouped with the generic type?

Reply:
That's totally fine for me as well.

Maybe you can make that correction if that's alright? I went on vacation now, so won't have time until January. I still hope this is a help in the right direction.

Reply:
I'll do that. Happy holidays!